src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(x) {
+    count++;
+
+    return (y) => {
+      if (count > 3 && count % 2 === 0) {
+        return `Bzzz... Error!`;
+      } else {
+        return x + y;
+      }
```

Comment:
```suggestion
      }
      
      return x + y;
```
else is redundant here, you can go straight to return.

Reply:
agree, remove else